src/main/java/ua/bank/bankservice/util/CurrencyConverter.java

```diff
@@ -0,0 +1,25 @@
+package ua.bank.bankservice.util;
+
+import com.google.gson.JsonObject;
+import java.math.BigDecimal;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import ua.bank.bankservice.model.Currency;
+
+@Component
+public class CurrencyConverter {
+    @Autowired
+    private HttpClient client;
+    @Value(value = "${url.converter}")
+    private String baseUrl;
+
+    public BigDecimal convert(Currency fromCurrency, Currency toCurrency, int amount) {
```

Comment:
Maybe it's better to pass BigDecimal?
```suggestion
    public BigDecimal convert(Currency fromCurrency, Currency toCurrency, BigDecimal amount) {
```